### PR TITLE
fix: Fix chectl logs before che pods start

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### What does this PR do?
Sometimes chectl fails before Che pods start and that means there are not any logs. 
This PR check if directory logs are empty and print a message.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-859

### CODE REVIEW

**Comment by flacatus** [comment]
image

**Comment by flacatus** [comment]
image

File: src/commands/server/start.ts [hunk 1]
```diff
@@ -402,6 +414,10 @@ export default class Start extends Command {
       await postInstallTasks.run(ctx)
       this.log('Command server:start has completed successfully.')
     } catch (err) {
+      const isEmptyDir = await this.isDirEmpty(ctx.directory)
+      if (isEmptyDir) {
+        this.error('There are no any logs available for the current installation')
```
Comment (tolusha) [inline src/commands/server/start.ts:419]: `this.error(`${err}\nInstallation failed, there are no available logs.`)`

Let's remove this line 
https://github.com/che-incubator/chectl/blob/master/src/commands/server/start.ts#L386

**Comment by dmytro-ndp** [comment]
IMHO "Installation failed, check logs in '${ctx.directory}'`)" message should be shorten if there are no logs available.

File: src/commands/server/start.ts (hunk 1 above)
Comment (flacatus) [reply src/commands/server/start.ts:419]: If `this.error(${err}\nInstallation failed, there are no available logs.)` The logs looks ugly now:

Example: `Error: Error: Platform is required ¯\_(ツ)_/¯`

File: src/commands/server/start.ts [hunk 2]
```diff
@@ -402,6 +413,10 @@ export default class Start extends Command {
       await postInstallTasks.run(ctx)
       this.log('Command server:start has completed successfully.')
     } catch (err) {
+      const isEmptyDir = await this.isDirEmpty(ctx.directory)
+      if (isEmptyDir) {
+        this.error(`${err}\nThere are no any logs available for the current installation`)
+      }
       this.error(`${err}\nInstallation failed, check logs in '${ctx.directory}'`)
```
Comment (dmytro-ndp) [inline src/commands/server/start.ts:420]: `check logs in '${ctx.directory}` doesn't make much sense if there are no any logs available.

File: src/commands/server/start.ts [hunk 3]
```diff
@@ -416,7 +415,7 @@ export default class Start extends Command {
     } catch (err) {
       const isEmptyDir = await this.isDirEmpty(ctx.directory)
       if (isEmptyDir) {
-        this.error('There are no any logs available for the current installation')
+        this.error(`${err}\nThere are no any logs available for the current installation`)
```
Comment (dmytro-ndp) [inline src/commands/server/start.ts:418]: Shouldn't we write something about the fact that installation failed? Just empty installation logs couldn't necessary mean installation failed, IMHO.

Comment (dmytro-ndp) [reply src/commands/server/start.ts:418]: The logic could be the following:
```
if (isEmptyDir) {
     this.error(`${err}\nInstallation failed. No logs available.`)
} else {
     this.error(`${err}\nInstallation failed, check logs in '${ctx.directory}'`)
}
```

Comment (tolusha) [reply src/commands/server/start.ts:418]: Actually `this.error(): never`. 
It terminates the program. The logic with `if` isn't needed.

Comment (tolusha) [reply src/commands/server/start.ts:418]:
```
if (isEmptyDir) {
     this.error(`${err}\nInstallation failed. There are no available logs.`)
}
this.error(`${err}\nInstallation failed, check logs in '${ctx.directory}'`)

**Review by tolusha — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**